metaflow/plugins/aip/aip.py

```diff
@@ -286,8 +286,23 @@ def _create_workflow_yaml(
         for key, value in self._get_flow_labels().items():
             workflow["metadata"]["labels"][key] = value
 
+        KubeflowPipelines._add_archive_section_to_cards_artifacts(workflow)
+
         return workflow
 
+    @staticmethod
+    def _add_archive_section_to_cards_artifacts(workflow: dict):
+        print("in _add_archive_section_to_cards_artifacts")
```

Comment:
Just a reminder to remove all debugging print, or update them to make more sense to users.